src/develop/ui/patterns/mobile/datepicker.md

```diff
@@ -3,68 +3,79 @@ tags: runtime-mobileandreactiveweb;
 summary: 
 ---
 
-# DatePicker Pattern
+# Date Picker
 
-The DatePicker pattern provides you with a date and time picker with a flat UI to display inline on the screen. It can receive lists of dates with events and it enables you have a selection within a range of days. The DatePicker was created using the [Pikaday.js
-library](https://github.com/dbushell/Pikaday/blob/master/README.md "https://github.com/dbushell/Pikaday/blob/master/README.md") .
+You can use the Date Picker UI pattern as a graphical interface widget that allows the user to select a date from a calendar and/or time from a time range.
+
+To display items in a gallery pattern, you can use static data or a **List** widget inside the block. 
+
+![](images/datepickercalendar.png)  
 
-You can use this pattern to display a list of elements side by side, with a different number of items per row on different devices.
+The Date Picker UI pattern was created using the [Pikaday.js
+library](https://github.com/dbushell/Pikaday/blob/master/README.md "https://github.com/dbushell/Pikaday/blob/master/README.md") .
 
-## How to Use the DatePicker Pattern
 
-Use static data or a **List** widget inside this block to display items in a gallery pattern.
+## How to use the Date Picker UI Pattern
 
-![](images/datepicker.png?width=600)  
+1. In Service Studio, in the Toolbox, search for `Date Picker`.
+  
+     The Date Picker widget is displayed.
+   
+    ![](images/datepicker-widget.png)  
+  
+1. From the Toolbox, drag the Date Picker widget onto your application's screen.
 
-1\. Upon dragging the DatePicker to the page, you'll be prompted to create an
-event.
+    You are prompted to create an event.
```

Comment:
This is innacurate. We are not prompted after dragging the widget to the screen.

Please check the previous steps:
https://success.outsystems.com/Documentation/11/Developing_an_Application/Design_UI/Patterns/Using_Traditional_Web_Patterns/Controls/DatePicker

And try to go over the steps one by one.

Most of the ones in the previous document were good - certainly a bit more detail in the "Create the handler events" would be helpful, but the current steps we have here are not correct.